### ISSUE TITLE
Deleting last Title in Favorites causes the application to crash 

### ISSUE THREAD

**Issue by InsanePrimate** [jimbob4000/RetroFlow-Launcher#119]
As the title says, the application crashes consistently if the last Title is deleted and the favorites category becomes empty. I don't even know if it is worth fixing, since the implementation of collections makes the favorites function redundant. I would just get rid of it. Everyone who wants a favorites category can now simple create a collection and name it Favorites... just a suggestion though XD

**Comment by jimbob4000** [comment]
Thanks for reporting, this has been fixed in the nightly build:
https://github.com/jimbob4000/RetroFlow-Launcher/releases/tag/Nightly

I'm keen to keep favourites, some people may not use collections, plus the heart icon is handy for seeing at a glance whether you have tagged it.

**Comment by InsanePrimate** [comment]
True... an Icon to indicate whether a game is part of a collection would also be nice... just a suggestion though :-)
